Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM mhart/alpine-node:base
+
+ENV PATH /root/.yarn/bin:$PATH
+
+RUN apk update \
+  && apk add curl bash binutils tar git \
+  && rm -rf /var/cache/apk/* \
+  && /bin/bash \
+  && touch ~/.bashrc \
+  && curl -o- -L https://yarnpkg.com/install.sh | bash \
+  && git clone https://github.com/rage8885/OpenSC2K \
+  && cd OpenSC2K \
+  && yarn --ignore-engines install \
+  && echo 'cd /OpenSC2K && yarn run start' >/run.sh \
+  && chmod +x /run.sh \
+  && apk del git curl tar binutils
+  
+EXPOSE 3000
+
+ENTRYPOINT [ "/bin/bash" ]
+
+CMD [ "/run.sh" ]
```

Comment:
If adding the files to the `/home/node/app` in the node:8+ base images, the working directory should be `/home/node/app`. Essentially all that would need to be run (after the container is built) is `yarn run start`. That would look like `CMD ["yarn", "run", "start"]`.